translations/en-us.yml

```diff
@@ -205,6 +205,7 @@ dashboard:
 
 search:
     page-title: 'Search'
+    institutions-title: 'Search Institutions'
```

Comment:
where is this translation string used?

Reply:
This is not being used currently, it is now removed.